tests/test_completion.py

```diff
@@ -206,6 +206,14 @@ def test_inputless_commands(hlwm, name):
         .returncode == 7
 
 
+def test_completionless_commands(hlwm):
+    # commands that accept arguments but don't have compltion.
```

Comment:
typo completion

Reply:
Was this even intended to be in this change? The commit message only subscribes removal of unused code.

Reply:
I've fixed the typo.

Yes this is intended. It tests that the code removal in `complete_against_commands()` does not break anything (`spawn` and `wmexec` are still using the C-style command interface)

Reply:
Thanks for the fix and adding the info to the commit message :)